src/gui/components/navbar/navbar.styles.js

```diff
@@ -0,0 +1,15 @@
+import { css } from 'glamor'
+import { primaryColor } from '../../styles'
+
+const layout = css({
+  display: 'flex',
+  alignItems: 'center',
+})
+
+export const title = css({
+  fontSize: '1.2em',
+  fontWeight: 'bolder',
+  paddingLeft: '10px',
+})
+
+export default css(primaryColor, layout)
```

Comment:
It seems we can [compose / condition](https://github.com/threepointone/glamor/blob/master/docs/composition.md) glamor css like that for example :
```javascript
compose([default, primary ? primaryColor : secondaryColor])
```